org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/DebriefLiteApp.java

```diff
@@ -466,7 +466,8 @@ private static void setInitialArea(final JMapPane mapPane,
       Application.logError2(ToolParent.ERROR,
           "Failure in setting initial viewport coverage", e);
     }
-    mapPane.getMapContent().getViewport().setFixedBoundsOnResize(true);
+    //TODO:geo tools update ???
+    //mapPane.getMapContent().getViewport().setFixedBoundsOnResize(true);
```

Comment:
this method was inserted, just for us, in this PR:
https://github.com/geotools/geotools/pull/2324/files

I _think_ we may need to switch to GeoTools 21.2 for it to be in the release.